armi/physics/neutronics/latticePhysics/tests/test_latticeInterface.py

```diff
@@ -19,38 +19,71 @@
 from armi.physics.neutronics.latticePhysics.latticePhysicsInterface import (
     LatticePhysicsInterface,
 )
-from armi.reactor.tests.test_reactors import loadTestReactor
-from armi.settings import Settings
-
+from armi import settings
+from armi.settings.fwSettings.globalSettings import CONF_RUN_TYPE
+from armi.operators.operator import Operator
+from armi.reactor.reactors import Reactor, Core
+from armi.physics.neutronics.crossSectionGroupManager import CrossSectionGroupManager
 
 # As an interface, LatticePhysicsInterface must be subclassed to be used
 class LatticeInterfaceTester(LatticePhysicsInterface):
     def __init__(self, r, cs):
         self.name = "LatticeInterfaceTester"
-        super(LatticeInterfaceTester, self).__init__(r, cs)
+        super().__init__(r, cs)
 
     def _getExecutablePath(self):
         return "/tmp/fake_path"
 
     def readExistingXSLibraries(self, cycle):
         pass
 
+    def _newLibraryShouldBeCreated(self, cycle, representativeBlockList, xsIDs):
+        return False
+
 
 class TestLatticePhysicsInterface(unittest.TestCase):
     """Test Lattice Physics Interface."""
 
+    @classmethod
+    def setUpClass(cls):
+        cls.o = Operator(settings.Settings())
+        cls.o.r = Reactor("empty", None)
+        cls.o.r.core = Core("empty")
+        xsGroupInterface = CrossSectionGroupManager(cls.o.r, cls.o.cs)
+        cls.latticeInterface = LatticeInterfaceTester(cls.o.r, cls.o.cs)
+        cls.o.addInterface(xsGroupInterface)
+        cls.o.addInterface(cls.latticeInterface)
+
+    def setUp(self):
+        self.o.r.core.lib = "Nonesense"
```

Comment:
not important, but
```suggestion
        self.o.r.core.lib = "Nonsense"
```
yay or nay?